test/lambda/ca-registrator/certificate-generator.test.ts

```diff
@@ -51,39 +52,61 @@ describe('Call function getCaRegistrationCertificates', () => {
 });
 
 describe('Call function getCaRegistrationCertificates without CSR subjects', () => {
+
+  let caCertificates: cg.CaRegistrationRequiredCertificates;
+
+  beforeEach(() => {
+    caCertificates = cg.getCaRegistrationCertificates();
+  });
+
   test('Certificates information are defined', () => {
-    var certificates = cg.getCaRegistrationCertificates();
-    expect(typeof certificates.ca.privateKey).toBe(typeof '');
-    expect(typeof certificates.ca.publicKey).toBe(typeof '');
-    expect(typeof certificates.ca.certificate).toBe(typeof '');
-    expect(typeof certificates.verification.privateKey).toBe(typeof '');
-    expect(typeof certificates.verification.publicKey).toBe(typeof '');
-    expect(typeof certificates.verification.certificate).toBe(typeof '');
+    expect(typeof caCertificates.ca.privateKey).toBe(typeof '');
```

Comment:
isString() toEqual true

Reply:
Do you mean util.isString() in Node JS?
It has been deprecated since version 4.0.0.
See: https://nodejs.org/api/util.html#util_util_isstring_object
IDE Prompt: @deprecated — since v4.0.0 - use typeof value === 'string' instead.

Reply:
const value1 = 'ABC';
  const value2 = 123;
  expect(typeof value1).toBe('string'); // true
  expect(typeof value2).toBe('string'); // false
  expect(value1).toEqual(expect.any(String)); // true
  expect(value2).toEqual(expect.any(String)); // false